admin/google_search_console/class-crawl-issue.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @package WPSEO\Admin|Google_Search_Console
+ */
+
+/**
+ * Class WPSEO_Crawl_Issue
+ *
+ * @todo Might want to create a class that accepts the raw Google response as construct parameter.
```

Comment:
Will remove this line, it's from the previous GWT implementation. It's not the task of this object to parse response. It should be parse before and given well formatted to this object.